torchvision/models/mobilenetv2.py

```diff
@@ -17,24 +16,6 @@
 
 
 # necessary for backwards compatibility
-class _DeprecatedConvBNAct(Conv2dNormActivation):
-    def __init__(self, *args, **kwargs):
-        warnings.warn(
-            "The ConvBNReLU/ConvBNActivation classes are deprecated since 0.12 and will be removed in 0.14. "
-            "Use torchvision.ops.misc.Conv2dNormActivation instead.",
-            FutureWarning,
-        )
-        if kwargs.get("norm_layer", None) is None:
-            kwargs["norm_layer"] = nn.BatchNorm2d
-        if kwargs.get("activation_layer", None) is None:
-            kwargs["activation_layer"] = nn.ReLU6
-        super().__init__(*args, **kwargs)
-
-
-ConvBNReLU = _DeprecatedConvBNAct
-ConvBNActivation = _DeprecatedConvBNAct
```

Comment:
@YosuaMichael Just being paranoid. Have you confirmed that this works on FBCode and that there are no breakages?

Reply:
Hi @datumbox, I have tested on this diffs: [D39730821](https://www.internalfb.com/diff/D39730821). It is a stack on top of the diffs that remove everything including `transforms/_functional_video.py` but I add this file back.